www_src/pages/page/page.less

```diff
@@ -38,30 +38,41 @@
     }
   }
 
-  .overlay {
+  .overlay-enter {
     position: fixed;
     top: 0;
     left: 0;
     height: 100%;
     width: 100%;
-    z-index: 998;
+    z-index: 999;
```

Comment:
bizar as it seems, this is not high enough. Let's use 2147483647, the maxint value allowed for zindex, with a comment that explains WHY it's this high.